iron-ajax.html

```diff
@@ -352,10 +359,10 @@
     /**
      * Performs an AJAX request to the specified URL.
      *
-     * @return {iron-request}
+     * @return {!IronRequestElement}
      */
     generateRequest: function() {
-      var request = document.createElement('iron-request');
+      var request = /** @type {!IronRequestElement} */ (document.createElement('iron-request'));
```

Comment:
Thanks for the explanation. Typing sounds useful in the case you described (and surely others), but I find trying to remember the appropriate annotations and correct usage is hard. I imagine it's harder than using a typed language would be.